rosys/persistence/persistable.py

```diff
@@ -31,6 +32,7 @@ def persistent(self, *,
                    restore: bool = True,
                    backup_check_interval: float | None = 10.0,
                    backup_interval: float | None = None,
+                   allow_tests: bool = False,
```

Comment:
```suggestion
                   allow_in_tests: bool = False,
```
Maybe `allow_in_tests` is a better name?